src/System.IO.Ports/tests/SerialPort/Read_char_int_int.cs

```diff
@@ -356,10 +355,7 @@ public void GreedyRead()
                 System.Text.Encoding.UTF32.GetChars(byteBuffer, 0, byteBuffer.Length, expectedChars, 1);
                 var charRcvBuffer = new char[(int)(expectedChars.Length * 1.5)];
 
-                while (com1.BytesToRead < 4 + numBytes)
-                {
-                    System.Threading.Thread.Sleep(50);
-                }
+                TCSupport.WaitForReadBufferToLoad(com1, 4+numBytes);
```

Comment:
Nit spacing around +. I won't mention this elsewhere. A run through https://github.com/dotnet/codeformatter should fix that.